.clusterstack.yaml.example

```diff
@@ -0,0 +1,30 @@
+apiVersion: infrastructure.clusterstack.x-k8s.io/v1alpha1
```

Comment:
repository already contains a directory called `examples`, see https://github.com/SovereignCloudStack/cluster-stack-provider-openstack/tree/main/examples

Consider removing the mentioned directory and then also provide an example of the `Cluster` resource in the root dir. 
Alternatively, move and merge this file there.

Reply:
I've included it inside examples directory, the clusterstack in examples directory doesn't mention `OpenStackClusterStackReleaseTemplate` because I wanted to keep it as a standalone clusterstack example. I understand they go hand in hand but the other combined one is already there in `config/cspo/clusterstack.yaml`

Reply:
IMO it is fine now, thank you!